e2e/pages/app/badge/badge.testpage.component.html

```diff
@@ -0,0 +1,92 @@
+<div class="badge-container">
+    <!-- icon example no content -->
+    <ul class="element-list">
+        <ng-container *ngFor="let vertPos of verticalPositions">
+            <ng-container *ngFor="let horzPos of horizontalPositions">
+                <li>
+                    <ux-icon name="filter" uxBadge badgeColor="warning" badgeVerticalPosition="{{ vertPos }}"
```

Comment:
Could we update any attributes using interpolation, eg:

```html
badgeVerticalPosition="{{ vertPos }}"
```

Should be:

```html
[badgeVerticalPosition]="vertPos"
```

While technically this will work for any string values it will not for any other type and we always recommend using property binding instead.